FeedStoreChallenge/CacheImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CacheImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Livia Vasconcelos on 19/06/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(CacheImage)
+internal class CacheImage: NSManagedObject {
```

Comment:
This class name can be confusing, since it can be understood as "One image that represents one specific `Cache` object" (similar to how you would read "UserImage", for example). To better clarify it's meaning I suggest changing it to `CachedImage`

Reply:
The convention in this codebase is to omit the default access level keyword of `internal`, please remove it from this class and its properties and methods.
Also, `CacheImage` is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class CacheImage: NSManagedObject {
```

Reply:
Renamed and fixed!

Reply:
Regarding the `internal` keyword, I believe you missed the part where I said "please remove it from this class **and its properties and methods**."
I preferred to not make a gigantic suggestion block with all of them, but I should have been clearer.